CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v0.0.5
+
+## Added
+* Included functions for simpler model indexing for sensors and joints
```

Comment:
1. put this change under v0.0.4 (which hasn't released yet, we should clarify that the top one is under development in the CHANGELOG) + delete v0.0.5 header
2. add `(@bhung-bdai, #76)` to this line to indicate who contributed and where